library/build.gradle.kts

```diff
@@ -6,17 +6,27 @@ plugins {
     id("com.vanniktech.maven.publish") version "0.27.0"
 }
 
+publishing {
+    repositories {
+        maven {
+            name = "AuthSDK"
+            url = uri("https://aws.oss.sonatype.org/service/local/staging/deploy/maven2/")
+            credentials(PasswordCredentials::class)
+        }
+    }
+}
+
 mavenPublishing {
     publishToMavenCentral(SonatypeHost.DEFAULT, automaticRelease = true)
     signAllPublications()
 
-    coordinates("software.amazon.location", "auth", "0.0.2")
+    coordinates("software.amazon.location", "auth", "0.2.4")
```

Comment:
Will it publish a new version only when we bump the version here?

Reply:
"Release" in this case is a GitHub-specific term, and it can only be triggered through the GitHub UI or CLI. The version number is not directly related.

Reply:
Is there a way for you to query maven if the version you're trying to publish is already released? Here's what we have for an npm repo:

https://github.com/aws-geospatial/amazon-location-for-maplibre-gl-geocoder/pull/94

Reply:
I will look into that query - it is possible on "normal" Maven, but we may have to adjust it for our unique situation.